easy_manipulation_deployment/grasp_planner/include/grasp_planner/common/math_functions.hpp

```diff
@@ -24,8 +24,30 @@
 #include "boost/uuid/random_generator.hpp"
 #include "boost/uuid/uuid_io.hpp"
 
+#include <pcl/point_types.h>
 namespace MathFunctions
 {
+
+struct Point
+{
+  Eigen::Vector3f reference_vector;
+  Eigen::Vector3f reference_point;
+  float projection_distance;
+  float perpendicular_distance;
+  int index;
+  pcl::PointNormal pcl_npoint;
+  Point(
+    Eigen::Vector3f reference_vector_,
+    Eigen::Vector3f reference_point_,
+    float projection_distance_,
+    float perpendicular_distance_)
+  {
+    reference_vector = reference_vector_;
```

Comment:
use initializer list

The reason is https://stackoverflow.com/questions/9903248/initializing-fields-in-constructor-initializer-list-vs-constructor-body


```
#include <iostream>

using namespace std;

struct TestStruct {
        int id;
        TestStruct() : id(42)
        {
        }
};

int main()
{
    TestStruct t;
    cout<<"Hello World" << t.id;

    return 0;
}

```

Reply:
Rectified. 75d3c9a